integration_tests/src/bin/benchmark/args.rs

```diff
@@ -99,6 +99,104 @@ impl From<EncoderPreset> for ffmpeg_h264::EncoderPreset {
     }
 }
 
+#[derive(Debug, Clone, Copy, clap::ValueEnum)]
+#[clap(rename_all = "snake_case")]
+pub enum ResolutionPreset {
+    Uhd,
+    Qhd,
+    Fhd,
+    Hd,
+    Sd,
+}
```

Comment:
I think it would be cool if the only thing you specified on the cmd was the number ++ p: `--resolution 1080p`. I would hate if you had to specify the resolution like this: `--resolution res_480_p`.

Also, `derive(ValueEnum)` does nothing here I think, since you implement parsing manually anyway.